provider/database.py

```diff
@@ -17,15 +17,30 @@
 import uuid
 
 from cloudant import Cloudant
+from cloudant import CouchDB
 from cloudant.result import Result
 
 class Database:
     db_prefix = os.getenv('DB_PREFIX', '')
     dbname = db_prefix + 'ow_kafka_triggers'
-    username = os.environ['CLOUDANT_USER']
-    password = os.environ['CLOUDANT_PASS']
+    username = os.environ.get('CLOUDANT_USER')
+    client = None
 
-    client = Cloudant(username, password, account=username)
+    if (username):
```

Comment:
Why not check the value of `DB_PROVIDER`? Above it seems odd to get the `CLOUDANT_USER` variable just to do this check. On first reading, I thought that `CLOUDANT_USER` was required. I understand that it is not required, but it made the code harder to understand.